package.json

```diff
@@ -53,4 +53,4 @@
     "tslint": "~5.11.0",
     "typescript": "~3.1.1"
   }
-}
\ No newline at end of file
+}
```

Comment:
@AlexandreInsua Can you please remove package.json and package-lock.json from the commit?